protocol/0017-PART-party.md

```diff
@@ -33,8 +33,12 @@ The `network` party is a pseudoparty. It is used in [position resolution](./0012
 # Acceptance Criteria
 
 1. [x] When a [standard party](#standard-party) must be uniquely identified, it must be identified by the public key of an ed25519 keypair (<a name="0017-PART-001" href="#0017-PART-001">0017-PART-001</a>)
+1. [x] A Party's transaction is rejected if the party has no balance in any asset - this includes proposals, voting and delegations (<a name="0017-PART-002" href="#0017-PART-002">0017-PART-002</a>)
```

Comment:
@davidsiska-vega  - please can this be re-worded but also looking at 0049-TVAL to avoid duplications